File: app/src/main/java/com/hitesh/todopod/itemsDAO.java
package com.hitesh.todopod;

import android.provider.ContactsContract;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;
import androidx.room.Update;

import java.util.List;

@Dao
public interface itemsDAO {

    @Insert
    void insert(items items);

    @Update
    void update(items items);

    @Delete
    void delete(items items);

    @Query("DELETE FROM note_table")
    void deleteAllNotes();

    @Query("SELECT * FROM note_table")
    LiveData<List<items>> getAllNotes();
}

File: app/src/main/java/com/hitesh/todopod/itemsDAO.kt
package com.hitesh.todopod

import androidx.lifecycle.LiveData
import androidx.room.*

@Dao
interface itemsDAO {
    @Insert
    fun insert(items: items?)

    @Update
    fun update(items: items?)

    @Delete
    fun delete(items: items?)

    @Query("DELETE FROM note_table")
    fun deleteAllNotes()

    @get:Query("SELECT * FROM note_table")
    val allNotes: LiveData<List<items>>
}